extensions/amp-layout/amp-layout.md

```diff
@@ -0,0 +1,55 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-layout"></a> `amp-layout`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>A generic, multi-purpose container element that can be used independent of other elements</td>
+  </tr>
+  <tr>
```

Comment:
BTW, is this component "stable", that is, already in production? Or experimental?  If not in production, please add an "availability" row with the availability status (similar to what [amp-viz-vega](https://github.com/ampproject/amphtml/blob/master/extensions/amp-viz-vega/amp-viz-vega.md) has): 
```html
  <tr>
    <td width="40%"><strong>Availability</strong></td>
    <td><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a></td>
  </tr>
```